### ISSUE TITLE
Password: Html validation errors (aria-readonly, role)

### ISSUE THREAD

**Issue by fcjm** [primefaces/primefaces#6672]
**Describe the defect**
I'm currently working on a project which was recently tested for valid html code. Among other components, we use the Primefaces password component which causes some errors when validating the html which is displayed in the browser. I have also tested it via primefaces-test for the 9.0-SNAPSHOT version, but the errors still exists.

The password field in the resulting html shows up with both "role='textbox'" and "aria-readonly='false'" attributes, which both cause error messages. Both of these attributes are set in the "core.js" -> "skinInput()" method.

Validation errors:
1. "Attribute role not allowed on element input at this point."
2. "Attribute aria-readonly not allowed on element input at this point."

**Environment:**
 - PF Version: _9.0-SNAPSHOT_
 - JSF + version: _Mojarra 2.3.14_
 - Affected browsers: _ALL_

**To Reproduce**
1. Go to e.g. 'http://localhost:8080/primefaces-test/'
2. Open e.g. Nu Html Checker
3. See error

**Example XHTML**
Input
```html
<p:password id="password-test" value="test"/>
```
Result
```html
<input id="frmTest:password-test" name="frmTest:password-test" type="password" class="ui-inputfield …i-widget ui-state-default ui-corner-all" role="textbox" aria-disabled="false" aria-readonly="false">
```

**Comment by melloware** [comment]
I ran the PF Password against Google LightHouse and it reports...
![image](https://user-images.githubusercontent.com/4399574/102347394-59c57f80-3f6e-11eb-9e63-cbf65e5c69a1.png)

What HTML validator are you using?

Although I did find this that says type="password" should have no role.
https://www.w3.org/TR/html-aria/


input type=password | No corresponding role | No roleGlobal aria-* attributes and aria-required
-- | -- | --


I will fix this.

**Comment by fcjm** [comment]
I am using the Nu Html Checker. The first bookmarklet on the site - https://validator.w3.org/nu/about.html#extras) - to be exact.

**Comment by melloware** [comment]
Thanks for reporting!  Fixed for 9.0

**Comment by fcjm** [comment]
Thank you for the quick answers and the fix! I have noticed that with this fix the "aria-disabled" won't be set as well. This one did not show up as an error though,

**Comment by melloware** [comment]
AHh OK let me make disabled OK.

**Comment by melloware** [comment]
OK I fixed it master to leave aria-disabled.
